desktop-widgets/tab-widgets/TabDiveSite.cpp

```diff
@@ -8,7 +8,10 @@ TabDiveSite::TabDiveSite(QWidget *parent) : TabBase(parent)
 {
 	ui.setupUi(this);
 	ui.diveSites->setTitle(tr("Dive sites"));
-	ui.diveSites->setModel(LocationInformationModel::instance());
+	ui.diveSites->setModel(&model);
```

Comment:
looking at just THIS commit, you add the private `model` and set that model, but I can't see us ever setting that member to anything useful (like `LocationInformationModel::instance()`)
This may be in a future commit, but right here this raises alarm bells

Reply:
As you noted below, `LocationInformationModel::instance()` is set as source model in `DiveSiteSortedModel`. This should be fine (and it appears to work).